tests/longevity-1TB-7days-authorization-and-tls-ssl.yaml

```diff
@@ -38,8 +38,8 @@ instance_provision: 'on_demand'
 # if running on aws and use_mgmt is true, the monitor image should not contain scylla
 use_mgmt: true
 mgmt_port: 10090
-scylla_repo_m: 'http://repositories.scylladb.com/scylla/repo/f4a2920f80c4bf178217c2553ad65ad7/centos/scylladb-2017.1.repo'
-scylla_mgmt_repo: 'http://downloads.scylladb.com.s3.amazonaws.com/manager/rpm/unstable/centos/branch-1.0/latest/scylla-manager.repo'
+scylla_repo_m: SCYLLA_ENTERPRISE_REPO
```

Comment:
This shouldn't be a variable here, because I don't want to pass it in all longevity jenkins jobs.
Please set it to the latest enterprise we have in the website (the official one):
http://repositories.scylladb.com/scylla/repo/TO_PUT_HERE_QA_REPO_ID/centos/scylladb-2018.1.repo

@amoskong, what is the repoid we should use so Amnon queries will ignore it?

Reply:
31df365 - Changed scylla_repo_m: scylladb-2018.1.repo , and - scylla_mgmt_repo: scylladb-manager-1.3.repo